### DISCUSSION TITLE
Creating an Empty Base Image

### DISCUSSION THREAD

**Discussion by adanvdo** [dlemstra/Magick.NET#1400] [Help]
My team is converting a project from nodejs to .net and I'm trying to find an image library that can support all of the image manipulation we are doing currently.  

Coming from node sharp, we were able to create an empty colored image with a designated number of channels and background color which we would then use to create a composite.

```
let base = await sharp({
    create: {
        width: plots.template.width,
        height: plots.template.height,
        channels: 4,
        background: this.hexColor
    }
})
```

I've read over the documentation but cannot seem to find an explanation or sample that shows this is possible. 
I did manage to find how to do it using Imagemagick, so it is supported by the software

`convert -size 800x800 canvas:white white.png`

Can someone help me with how to do this using Magick.NET?

**Comment by dlemstra** [comment]
You can create an empty canvas with this method:

```C#
using var image = new MagickImage(MagickColors.White, 800, 800);
```

Probably not relevant for you but if your project could do image manipulation on the client side you might also consider this library: https://github.com/dlemstra/magick-wasm.

**Reply by adanvdo** [reply]
Thank you for the quick response!
